README.md

```diff
@@ -18,7 +18,7 @@
 [![Maven Central](https://img.shields.io/maven-central/v/org.sonatype.nexus.plugins/nexus-repository-r.svg?label=Maven%20Central)](https://search.maven.org/search?q=g:%22org.sonatype.nexus.plugins%22%20AND%20a:%22nexus-repository-r%22)
 
 # Table Of Contents
-* [Release notes](#release-notes)
+* [Release notes](https://github.com/sonatype-nexus-community/nexus-repository-r/blob/master/RELEASE-NOTES.md)
```

Comment:
Q: did you intentionally not include 1.0.5 and 1.0.4 release notes?

Reply:
Minor: I suspect this does not have to be a hard coded link to /master.  See https://github.com/sonatype-nexus-community/nexus-repository-r/pull/109/files#diff-04c6e90faac2675aa89e2176d2eec7d8R38.

Reply:
19551623b6734262174d59b5fb45e2078d08a42a

Reply:
@joedragons  According to your first question, yes it was by design. Didn't include these versions because we have not such practice before. At the same time, there are no descriptions to add in these versions. most of the version was increased due to some small non-functional changes.

Reply:
Ok, then the only Q I have is if we intend to ever update the page again.  If we don't then the header being v1.10 seems overkill.  It could just be integrated into the text (e.g. "As of v1.10.0, ...").  I don't feel strongly about that though and will +1.

Reply:
I'm fine with have a link to release notes in Confluence.

Reply:
FWIW, I think the text currently on that page is worth having somewhere.  People who are using the plugin should be able to come and see it's now in product as of 1.1.0.  So if you change to just an h.s.c link I'd put somewhere else.